### ISSUE TITLE
'aria-roledescription' appears to be an incorrect aria label, should be 'role' ?

### ISSUE THREAD

**Issue by timvracer** [atlassian/react-beautiful-dnd#479]
## Bug or feature request?
Bug I believe

### Expected behavior
That fields utilizing drag/drop would conform to aria specifications and pass the AXE extension test for accessbility

### Actual behavior
Due to a tag named 'aria-roledescription', any fields using drag/drop are failing the AXE tool test for compliance.  I cannot find any such label in the aria documentation, and believe it should be 'role'

### Steps to reproduce
Run AXE on any page using Draggable's -- here is the div that was created:

`<div data-react-beautiful-dnd-draggable="0" tabindex="0" data-react-beautiful-dnd-drag-handle="0" aria-roledescription="Draggable item. Press space bar to lift" draggable="false" style="opacity: 1;"><div class="styles__HdrListItem-CnjcT ghSiCI"></div></div>`

**Comment by alexreardon** [comment]
We found we had the best screen reader experience not using 'role'. To get your tests passing you are welcome to add your own 'role'. for now. 

@seancurtis it looks like we could use a 'role' and potentially override it with 'aria-roledescription'. http://www.ets-research.org/ia11ylab/aria11spec/

Do you want to take a look?

**Comment by alexreardon** [comment]
We could add `role='button'` but this would prevent a draggable that is also a link from reading the correct? screen reader text for a draggable that is also a link as it looses the link information..

Perhaps we default to `role='button'` and let users opt into other roles as needed?

Thoughts @seancurtis ?

**Comment by gnapse** [comment]
I too spotted this when running a accessibility audit on a page using this library.

I get this from the audit:

> To solve this violation, you need to:
> Fix the following:
>
> Invalid ARIA attribute name: aria-roledescription

It does not seem to be a valid attribute, so I wonder why in practice it is giving a better screen reader experience? 🤔

**Comment by alexreardon** [comment]
It is super strange. I would be happy for suggestions!!! Right now using `role-description` gives the best screen reader experience

**Comment by gnapse** [comment]
I actual found the documentation for this attribute: https://www.w3.org/TR/wai-aria-1.1/#aria-roledescription so it does not seem to be invalid after all.

It is [deque's aXe](http://deque.com/axe/) chrome extension the one that makes the audit and flags this attribute as not valid.

So I dig up a little bit, and it is their issue, and it was raised in their repo: dequelabs/axe-core#234. However, they closed it because of not being widely supported. Take a look at the thread in that issue.

I think this could be closed then.

**Comment by hshyk** [comment]
The report generated by Lighthouse gives a similar error:
`[aria-*] attributes are not valid or misspelled`

Running React-axe seems to slow down the page on load considerably when there are many elements in a list (or list within a list). It combines the errors into one section, but logs each element to the console.

**Comment by alexreardon** [comment]
Related: #865

**Comment by alexreardon** [comment]
We will track this in #865
